ci/container/package.json

```diff
@@ -7,15 +7,21 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
+    "async": "3.2.2"
   },
   "devDependencies": {
-    "async": "^2.6.1",
+    "async": "3.2.2",
```

Comment:
Should this be greater than 3.2.2 instead of equal to 3.2.2?

Reply:
Setting to 3.2.3 had the same result

Reply:
No like set it to `"async": "^3.2.2",`

Reply:
Using `"async": "^3.2.2",` resulted in the same failure.

Replacing the `ocsp` package with a forked version that's using a later version of `async` seems to have fixed it